docs/OCP_CI_Tutorials/Scenarios/Scenarios_Guide.md

```diff
@@ -89,14 +101,17 @@ tests:
       BASE_DOMAIN: aws.interop.ccitredhat.com
     workflow: cucushift-installer-rehearse-aws-ipi
 ```
+
 A good way to verify your using what you expect to use for the value of cron you can use [Cronitor](https://crontab.guru/#0_1_*_*_1)
 
 ### Ensure Test execution output matches reporting
+
 In order for us to merge a MR we need to be sure the E2E testing is working. This includes reporting the test results.
 
 It is important that we verify the different levels of artifacts and make sure that they all align.
 
-For example we might need to check the following:
+For example, we might need to check the following:
+
 - Console output (prow job)
```

Comment:
```suggestion
- Console output (Prow job)
```

Reply:
Resolved in [**39fe241**7dca0aa2df5e3e0cb4676f4fa7ac99bf7](https://github.com/CSPI-QE/ocp-ci-docs/pull/10/commits/39fe2417dca0aa2df5e3e0cb4676f4fa7ac99bf7)